celestia/gentx/NakoTurk.txt

```diff
@@ -0,0 +1 @@
+1d824f2cf4df9c9da98c120b78be3f4319a1d7ff@46.4.23.42:26656
```

Comment:
@okannako this file needs to be in the peers directory in order for people to call collect gentx and in order for the CI to pass

Reply:
resolving as this file was moved

Reply:
Solved, Thank you